server/utils/s3.ts

```diff
@@ -20,7 +21,7 @@ type PubPubS3ClientConfig = {
 	accessKeyId: string;
 	secretAccessKey: string;
 	bucket: string;
-	ACL?: string;
+	ACL?: ObjectCannedACL;
```

Comment:
this is just a small type change, `tsc` complained about it when installing from the `sdk` repo.